src/spyglass/lfp/v1/lfp.py

```diff
@@ -164,15 +164,28 @@ def make(self, key):
                 "valid times",
             )
         )
-        IntervalList.insert1(
-            {
+
+        tmp_valid_times = (
+            IntervalList
+            & {
                 "nwb_file_name": key["nwb_file_name"],
                 "interval_list_name": key["interval_list_name"],
-                "valid_times": lfp_valid_times,
-                "pipeline": "lfp_v1",
-            },
-            replace=True,
-        )
+            }
+        ).fetch("valid_times")
+        if len(tmp_valid_times) == 0:
+            IntervalList.insert1(
+                {
+                    "nwb_file_name": key["nwb_file_name"],
+                    "interval_list_name": key["interval_list_name"],
+                    "valid_times": lfp_valid_times,
+                    "pipeline": "lfp_v1",
+                },
+                replace=True,
+            )
+        elif not np.allclose(tmp_valid_times[0], lfp_valid_times):
```

Comment:
Why is this compared to `tmp_valid_times[0]`? Isn't there sometimes an issue where the shape isn't (n_valid_times, 2) but instead (2,)?

Reply:
It's to pull out the list of interval times since tmp_valid_times comes from a `fetch` call instead of a `fetch1`
```
tmp_valid_times = (
            IntervalList
            & {
                "nwb_file_name": key["nwb_file_name"],
                "interval_list_name": key["interval_list_name"],
            }
        ).fetch("valid_times")
```

Reply:
Ah got it. Maybe slight preference for putting this up where you declare tmp_valid_times but it's not a big deal.